pgjdbc/src/test/java/org/postgresql/test/jdbc4/ArrayTest.java

```diff
@@ -75,6 +76,22 @@ public void tearDown() throws SQLException {
   }
 
   @Test
+  public void testCreateArrayOfBytea() throws SQLException {
+    PreparedStatement pstmt = _conn.prepareStatement("SELECT ?::bytea");
+
+    byte []b = new byte[] { (byte)0xde, (byte)0xad, (byte)0xbe, (byte)0xef }; // 0xdeadbeef
```

Comment:
I think it's time to use https://github.com/bradfitz/gitbrute in order to ensure the commit id starts with `deadbeef` as well. What do you think?